packages/amplify_datastore/ios/amplify_datastore.podspec

```diff
@@ -0,0 +1,23 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint amplify_datastore.podspec' to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'amplify_datastore'
+  s.version          = '0.0.1'
+  s.summary          = 'The DataStore module for Amplify Flutter.'
+  s.description      = <<-DESC
+The DataStore module for Amplify Flutter.
+                       DESC
+  s.homepage = 'https://github.com/aws-amplify/amplify-flutter'
+  s.license = 'Apache License, Version 2.0'
+  s.author = { 'Amazon Web Services' => 'amazonwebservices' }
+  s.source = { :git => 'https://github.com/aws-amplify/amplify-flutter.git' }
+  s.source_files = 'Classes/**/*'
+  s.dependency 'Flutter'
+  s.platform = :ios, '8.0'
```

Comment:
Now might be a good time to change s.platform to 13 (which I believe is the min version required)?

Reply:
changed